packages/@aws-cdk/aws-servicecatalogappregistry/test/integ.application-associator.cross-account-stack-enabled.ts

```diff
@@ -0,0 +1,26 @@
+import * as cdk from '@aws-cdk/core';
+import * as integ from '@aws-cdk/integ-tests';
+import * as appreg from '../lib';
+
+const app = new cdk.App();
+const localStack = new cdk.Stack(app, 'integ-servicecatalogappregistry-local-resource');
+
+new appreg.ApplicationAssociator(app, 'RegisterCdkApplication', {
+  applications: [appreg.TargetApplication.createApplicationStack({
+    enableCrossAccountStacks: true,
+    applicationName: 'AppRegistryAssociatedApplication',
+    stackName: 'TestAppRegistryApplicationStack',
+  })],
+});
+
+new cdk.Stack(app, 'integ-servicecatalogappregistry-cross-account-resource', {
+  env: {
+    account: '123456',
```

Comment:
While generating the snapshots, I used an actual account to complete the deployment successfully, and then replaced all places in the snapshot files where that account was mentioned with this dummy ID. Please let me know if this is an acceptable approach and/or if there is a better way to setup these integ test snapshots.

Reply:
in general we don't specify the `env` unless there's a specific region limitation for these integ test stacks

Reply:
If  you specify a fake id, then it won't deploy when someone attempts to deploy it. We should leave `env` unspecified here

Reply:
Got it, then I don't think there's a good way to test a true cross-account deployment scenario; instead I will make this focus on just one `env` while still exercising the code logic of handling cross-account stacks.